netbox_onboarding/netdev_keeper.py

```diff
@@ -257,27 +259,38 @@ def get_onboarding_facts(self):
 
             napalm_device.open()
 
-            logging.info("COLLECT: device facts")
+            logger.info("COLLECT: device facts")
             self.facts = napalm_device.get_facts()
 
-            logging.info("COLLECT: device interface IPs")
+            logger.info("COLLECT: device interface IPs")
             self.ip_ifs = napalm_device.get_interfaces_ip()
 
-            try:
-                module_name = PLUGIN_SETTINGS["onboarding_extensions_map"].get(self.napalm_driver)
+            module_name = PLUGIN_SETTINGS["onboarding_extensions_map"].get(self.napalm_driver)
+
+            if not module_name:
+                logger.info(
+                    "No onboarding extension defined for napalm driver %s, using default napalm driver",
+                    self.napalm_driver,
+                )
+
+            if module_name:
                 module = importlib.import_module(module_name)
                 driver_addon_class = module.OnboardingDriverExtensions(napalm_device=napalm_device)
```

Comment:
Would it make sense to refactor this block into it's own function? @mzbroch

Reply:
This way is more optimal:
```
if module_name:
...
elif not module_name:
```

Reply:
Why not just `if not module_name: .... else:`?

Reply:
Then `if module_name: ... else`, I rather prefer this order

Reply:
@mzb -- I've reformatted. My thought was more that evaluating whether or not to populate the `self.onboarding_class` and `self.driver_addon_result` falls a little bit outside the purview of `get_onboarding_facts`. I was thinking it might be good to give that it's own method?